libmachine/ssh/client.go

```diff
@@ -312,3 +356,34 @@ func (client ExternalClient) Shell(args ...string) error {
 
 	return cmd.Run()
 }
+
+func (client *ExternalClient) Start(command string) (io.ReadCloser, io.ReadCloser, error) {
+	args := append(client.BaseArgs, command)
+	cmd := getSSHCmd(client.BinaryPath, args...)
+
+	log.Debug(cmd)
+
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return nil, nil, err
+	}
+	stderr, err := cmd.StderrPipe()
+	if err != nil {
+		_ = stdout.Close()
```

Comment:
Why swallow the errors here?  At least aggregating them with `fmt.Errorf("%s, %s", err, closeErr)` seems reasonable, no?